app/main.py

```
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        command, source, destination = command.split()
+    except ValueError:
+        print("Three arguments expected")
```

use raise instead of print